src/app/models/transmart-models/transmart-cohort.ts

```diff
@@ -0,0 +1,23 @@
+import {CohortSubscriptionFrequency} from '../cohort-models/cohort-subscription-frequency';
+import {TransmartConstraint} from './transmart-constraint';
+
+export class TransmartCohort {
```

Comment:
Looks like a duplication of [GbBackendQuery](src/app/models/gb-backend-models/gb-backend-query.ts) class

Reply:
Ha, interesting. Wonder why there were two mappers in the first place then. I will merge them.

Reply:
There is an interesting difference:
In `GbBackendQuery`:
- `queryConstraint` (`object`)
- `queryBlob` (`object`)
In `TransmartCohort`:
- `constraint` (`TransmartConstraint`)